src/parser/paladin/retribution/modules/core/ArtOfWar.js

```diff
@@ -91,8 +91,8 @@ class AoWProcTracker extends Analyzer {
         position={STATISTIC_ORDER.OPTIONAL(2)}
         icon={<SpellIcon id={SPELLS.ART_OF_WAR.id} />}
         value={`${formatPercentage(this.consumedProcsPercent)}%`}
-        label="Art of War procs used"
-        tooltip={`You got ${this.totalAoWProcs} Art of War procs and used ${this.consumedAoWProcs} of them`}
+        label="Art of War Procs Used"
```

Comment:
Super nit pick but it's generally preferred to not Capitalize Every Single Word Of A Label As It Makes Reading Things Harder.